cfme/common/vm_views.py

```diff
@@ -0,0 +1,371 @@
+# -*- coding: utf-8 -*-
+from time import sleep
+
+from widgetastic.widget import View, Text, TextInput, Checkbox
+from widgetastic_patternfly import (
+    Dropdown, BootstrapSelect, FlashMessages, Tab, Input, BootstrapTreeview)
+
+from cfme.base.login import BaseLoggedInPage
+from cfme.exceptions import TemplateNotFound
+from widgetastic_manageiq import (
+    Calendar, SummaryTable, Button, ItemsToolBarViewSelector, Table, Search, MultiBoxSelect,
+    CheckableManageIQTree, VersionPick, Version, CheckboxSelect)
+
+
+class VMToolbar(View):
+    """
+    Toolbar view for vms/instances collection destinations
+    """
+    reload = Button(title='Reload current display')
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+    lifecycle = Dropdown('Lifecycle')
+    power = Dropdown('Power Operations')  # title
+    download = Dropdown('Download')
+
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class VMEntities(View):
+    """
+    Entities view for vms/instances collection destinations
+    """
+    title = Text('//div[@id="main-content"]//h1//span[@id="explorer_title_text"]')
+    adv_search_clear = Text('//div[@id="main-content"]//h1//span[@id="clear_search"]/a')
+    flash = FlashMessages('.//div[@id="flash_msg_div"]'
+                          '/div[@id="flash_text_div" or contains(@class, "flash_text_div")]')
+    search = View.nested(Search)
+    table = Table("//div[@id='list_grid']//table")
+
+
+class VMDetailsEntities(View):
+    """
+    Details entities view for vms/instances details destinations
+
+    VM's have 3-4 more tables, should inherit and add them there.
+    """
+    title = Text('//div[@id="main-content"]//h1//span[@id="explorer_title_text"]')
+    flash = FlashMessages('.//div[@id="flash_msg_div"]'
+                          '/div[@id="flash_text_div" or contains(@class, "flash_text_div")]')
+    properties = SummaryTable(title='Properties')
+    lifecycle = SummaryTable(title='Lifecycle')
+    relationships = SummaryTable(title='Relationships')
+    vmsafe = SummaryTable(title='VMsafe')
+    attributes = SummaryTable(title='Custom Attributes')  # Only displayed when assigned
+    compliance = SummaryTable(title='Compliance')
+    power_management = SummaryTable(title='Power Management')
+    security = SummaryTable(title='Security')
+    configuration = SummaryTable(title='Configuration')
+    diagnostics = SummaryTable(title='Diagnostics')
+    smart_management = SummaryTable(title='Smart Management')
+
+
+class ProvisionView(BaseLoggedInPage):
```

Comment:
whether this view isn't developed yet ? 
I guess we need to ask @lkhomenk about it.